project-progression-policy.md

```diff
@@ -118,15 +127,15 @@ The Graduation stage is for projects that have reached their growth goals and ar
 
 **Expectations**
 
-Graduation stage projects are “TAC Projects” under the Funding Charter of the Consortium, and as such have a voting representative on the TAC. They eligible to receive ongoing support from the Consortium as determined by the Governing Board, and are expected to cross promote the Consortium along with their activities.
+Graduation stage projects are "TAC Projects" under the Funding Charter of the Consortium, and as such have a voting representative on the TAC. They are eligible to receive ongoing support from the Consortium as determined by the Governing Board, and are expected to cross promote the Consortium along with their activities.
 
 **Acceptance Criteria**
 
 To graduate from Sandbox or Incubation status, or for a new project to join as an Graduation project, a project must meet the Incubation stage criteria plus:
 
  * Have a defined governing body of at least 5 or more members (owners and core maintainers, or similar technical role), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer. 
 * Have a healthy number of committers from at least two organizations. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
-* Adopt the a Code of Conduct.
+* Adopt a Code of Conduct.
```

Comment:
should this be adopt an _acceptable_ code of conduct?

Reply:
Only if we first agree on what "acceptable" means.   Since we haven't discussed that yet, I think this should be merged and that tracked via a separate issue.

Reply:
Created new issue #14 so as to decouple it from this PR.